.github/workflows/build.yaml

```diff
@@ -216,11 +216,13 @@ jobs:
       - name: Push latest rad cli binary to GHCR (unix-like)
         if: github.ref == 'refs/heads/main' && matrix.target_os != 'windows'
         run: |
-          oras push ${{ env.CONTAINER_REGISTRY }}/rad/${{ matrix.target_os }}-${{ matrix.target_arch }}:latest ./dist/${{ matrix.target_os}}_${{ matrix.target_arch}}/release/rad --annotation "org.opencontainers.image.source=${{ env.IMAGE_SRC }}"
+          cp ./dist/${{ matrix.target_os}}_${{ matrix.target_arch}}/release/rad ./rad
```

Comment:
We do not need to copy this binary to repo root. Instead, we can use `working-directory:`.

```
- name: Push latest rad cli binary to GHCR (unix-like)
        if: github.ref == 'refs/heads/main' && matrix.target_os != 'windows'
        working-directory: ./dist/${{ matrix.target_os}}_${{ matrix.target_arch}}/release
        run: |
             oras push ${{ env.CONTAINER_REGISTRY }}/rad/${{ matrix.target_os }}-${{ matrix.target_arch }}:latest ./rad --annotation "org.opencontainers.image.source=${{ env.IMAGE_SRC }}"
```